### PR TITLE
Removed use of pow()

### PR DESCRIPTION
**Pull request by DavidBel86**

### CODE REVIEW

**Comment by merbanan** [comment]
You have to rebase and squash this PR. There are merge conflicts.

File: src/devices/efergy_e2_classic.c [hunk 1]
```diff
@@ -60,24 +69,50 @@ static int efergy_e2_classic_callback(bitbuffer_t *bitbuffer) {
...
-	for (unsigned i = 0; VOLTAGES[i] != 0; ++i) {
-		double power  = (VOLTAGES[i] * current_adc * (1 << bytes[6])) / 32768;
-		fprintf(stderr, "Power consumption at %u volts: %.2f watts\n", VOLTAGES[i], power);
-	}
+	uint16_t address = bytes[2] << 8 | bytes[1];
+	uint8_t learn = (bytes[3] & 0x80) >> 7;
+	uint8_t interval = (((bytes[3] & 0x30) >> 4) + 1) * 6;
+	uint8_t battery = (bytes[3] & 0x40) >> 6;
+	uint8_t fact = (-(int8_t)bytes[6] + 15);
+	double current_adc = (double)((bytes[4] << 8 | bytes[5])) / (1 << fact);
```
Comment (merbanan) [inline src/devices/efergy_e2_classic.c]: Use (float) here. No need for double.

**Comment by merbanan** [comment]
Just one comment with regards to the code.

**Comment by DavidBel86** [comment]
Is it ok now? To be honnest, I'm not that familiar with the rebasing :P

**Comment by merbanan** [comment]
Squash worked.

**Pull request merged**